www-client/seamonkey/seamonkey-2.53.11.ebuild

```
@@ -0,0 +1,554 @@
+	jack? ( virtual/jack )
+	crypt? ( <x11-plugins/enigmail-2.1.0 )
+	kernel_linux? ( !pulseaudio? ( media-libs/alsa-lib ) )
+	pulseaudio? ( || (
+		media-sound/pulseaudio
+		>=media-sound/apulse-0.1.9
+	) )
+	startup-notification? ( >=x11-libs/startup-notification-0.8 )
+	system-av1? (
+		>=media-libs/dav1d-0.3.0:=
+		>=media-libs/libaom-1.0.0:=
+	)
+	system-harfbuzz? (
+		>=media-gfx/graphite2-1.3.9-r1
+		>=media-libs/harfbuzz-1.3.3:0=
+	)
+	system-icu? ( >=dev-libs/icu-59.1:= )
+	system-jpeg? ( >=media-libs/libjpeg-turbo-1.2.1 )
+	system-libevent? ( >=dev-libs/libevent-2.0:0= )
+	system-libvpx? ( >=media-libs/libvpx-1.8.0:0=[postproc] )
+	system-png? ( >=media-libs/libpng-1.6.31:0=[apng] )
+	system-sqlite? ( >=dev-db/sqlite-3.36.0:3[secure-delete,debug=] )
+	wifi? (
+		kernel_linux? (
+			>=dev-libs/dbus-glib-0.72
+			net-misc/networkmanager
+			>=sys-apps/dbus-0.60
+		)
+	)
+"
+RDEPEND="${COMMON_DEPEND}
+	selinux? ( sec-policy/selinux-mozilla )
+"
+DEPEND="${COMMON_DEPEND}
+	amd64? ( virtual/opengl )
+	x86? ( virtual/opengl )
+"
+
+# allow GMP_PLUGIN_LIST to be set in an eclass or
+# overridden in the enviromnent (advanced hackers only)
+[[ -z ${GMP_PLUGIN_LIST} ]] && GMP_PLUGIN_LIST=( gmp-gmpopenh264 gmp-widevinecdm )
+
+BUILD_OBJ_DIR="${S}/seamonk"
+
+pkg_setup() {
+	if [[ ${PV} == *_beta* ]] || [[ ${PV} == *_pre* ]] ; then
+		ewarn
+		ewarn "You're using an unofficial release of ${PN}. Don't file any bug in"
+		ewarn "Gentoo's Bugtracker against this package in case it breaks for you."
+		ewarn "Those belong to upstream: https://bugzilla.mozilla.org"
+	fi
+
+	if use crypt ; then
+		ewarn
+		ewarn "Enigmail has dropped support for Seamonkey in early 2019. If you are still"
+		ewarn "using it, consider yourself lucky, but also consider to start migrating away"
+		ewarn "from it. The crypt USE flag and its x11-plugins/enigmail dependency will be"
+		ewarn "removed in the near future."
+		ewarn
+	fi
+
+	moz_pkgsetup
+}
+
+pkg_pretend() {
+	# Ensure we have enough disk space to compile
+	if use debug || use lto || use test ; then
+		CHECKREQS_DISK_BUILD="16G"
+	else
+		CHECKREQS_DISK_BUILD="12G"
+	fi
+	check-reqs_pkg_setup
+}
+
+spkg_setup() {
+	# Ensure we have enough disk space to compile
+	if use debug || use lto || use test ; then
+		CHECKREQS_DISK_BUILD="16G"
+	else
+		CHECKREQS_DISK_BUILD="12G"
+	fi
+	check-reqs_pkg_setup
+}
+
+src_unpack() {
+	local l10n_sources="${P}.source-l10n.tar.xz"
+	unpack ${A/ ${l10n_sources}}
+
+	mkdir "${S}/${P}-l10n" || die
+	cd "${S}/${P}-l10n" || die
+	unpack ${l10n_sources}
+}
+
+src_prepare() {
+	# Apply our patches
+	eapply "${WORKDIR}"/gentoo-${PN}-patches-${PV}-r1/${PN}
+
+	# Shell scripts sometimes contain DOS line endings; bug 391889
+	grep -rlZ --include="*.sh" $'\r$' . |
+	while read -r -d $'\0' file ; do
+		einfo edos2unix "${file}"
+		edos2unix "${file}"
+	done
+
+	use system-libvpx \
+		&& eapply -p2 ""${WORKDIR}"/gentoo-${PN}-patches-${PV}-r1/USE_flag/1009_seamonkey-2.53.3-system_libvpx-1.8.patch"
```

Nit (I can fix this on merge):
```suggestion
		&& eapply -p2 "${WORKDIR}"/gentoo-${PN}-patches-${PV}-r1/USE_flag/1009_seamonkey-2.53.3-system_libvpx-1.8.patch
```

---

If there are no other objections, please do. While you're at it, please do the suggestion by juippis below (regarding moving the && to the line above) as well.

---

ack, thanks juippis!